developer/classic.md

```diff
@@ -148,7 +146,7 @@ const gtcr = new GeneralizedTCR(
   }
 ```
 
-Example [https://codesandbox.io/s/kind-banach-4fxnj](https://codesandbox.io/s/kind-banach-4fxnj)
+Example: [https://codesandbox.io/s/kind-banach-4fxnj](https://codesandbox.io/s/kind-banach-4fxnj)
```

Comment:
The codesandbox code fails to run

Reply:
No idea what code was running there, removed the link.